README.md

```diff
@@ -1 +1,49 @@
-# Portfolio-project
\ No newline at end of file
+# Portfolio-project
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portofolio
+
+## Description
+
+> This is my  portofolio  project which discribe my qualification. And here you found my recent works.  
+
+- ## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+1. Download the `Zip` or `git clone https://github.com/BouaikIlham/Portfolio-project.git`
+2. `cd Hello-Microverse`
+3. Then open in your browser by using `live server`
+
+## Authors
+
+👤 **Ilham Bouaik**
+
+
+-  GitHub : [@bouaik ilham](https://github.com/BouaikIlham)
+- Twitter: [@IlhamBouaik](https://twitter.com/IlhamBouaik)
+- LinkedIn: [@BouaikIlham](https://www.linkedin.com/in/bouaik-ilham-478478230/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Thank you Microverse
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] This `MIT` link is broken. Kindly fix this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.